SPECS/R/R.spec

```diff
@@ -0,0 +1,125 @@
+%global debug_package %{nil}
+
+Summary:        A language for data analysis and graphics
+Name:           R
+Version:        4.1.0
+Release:        2%{?dist}
+License:        GPLv2
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Daemons
+URL:            https://www.r-project.org
+Source0:        https://cran.r-project.org/src/base/R-4/R-%{version}.tar.gz
+
+BuildRequires:  build-essential
```

Comment:
>BuildRequires [](http://example.com/codeflow?start=0&length=13)

Nit: I'd suggest running the spec linter against all new specs. Just watch out for the `%{_lib}` macro!:) We need to fix that, whatever the linster changes to `%{_lib}` should most probably actually be `%{_libdir}`.